apps/calsys/values-tucson-teststand.yaml

```diff
@@ -3,5 +3,8 @@ namespace: calsys
 cscs:
```

Comment:
Rename this directory `calsys-lab` and change any references to that within the files found in this directory. Remove the GC stuff including the `renameMap`.

Reply:
You mean copy the folder and rename it?